### ISSUE TITLE
Install documentation

### ISSUE THREAD

**Issue by vielmetti** [psvensson/nrfe-widgets#6]
It would be nifty to include links or copies of whatever current installation documents that you have, as well as the "how to" for updating your installation if you are working from this Github repo rather than the main distribution or from a fork rather than your main distribution.  This might also include a link somewhere to your blog posts on the topic.

**Comment by psvensson** [comment]
I agree. I need to write a guide one can follow when installing everything
from the github repos and how to work with the repos together. I'll try to
do something proper thus week.
Shortly, though:

1. Fork my fork of the Evothings studio, which contains a little bit of
logic to start an embedded version of Node-red:
https://github.com/psvensson/evothings-studio/tree/node-red
2. Do the same for nrfe-widgets (https://github.com/psvensson/nrfe-widgets),
nrfe (https://github.com/psvensson/nrfe) and nrmodules (
https://github.com/psvensson/nrmodules)
3. Follow the README for evothings studio on how to initialize and build it
initially (download node-webkit, et.c.)
3.5 Also run npm install
4. Edit package.json for the studio so that "toolbar": true
5. Start the studio by running ./run.rb
6. Check out the nrfe hellow world project:
https://github.com/psvensson/nrfe-hello-world or TI Sensortag project:
https://github.com/psvensson/nrfe-ti-sensortag
7. Drag the index.html from the project onto the studio window. This will
add the whole project to the studio under 'My Apps'

The nrfe hello world project(s) contain their own copies of nrfe and
nrfe-widgets. When adding new widgets (the nrfe parser should probably be
fine for a while, hopefully), just run ./build.sh from the nrfe-widget
directory and copy the resulting index.js file into the hello
world/bower_components/nrfe-widgets directory.  "Official" releases, e.g.
by me can be fetched by doing 'bower update nrfe-widgets'

The nrmodules npm project is included in the node-red branch of
evothings-studio, so when adding new module editors to nrmodules, then copy
over the new files into the evothings-studio/node_modules/nrmodules
directory, and then restart the studio
